code/modules/mob/living/simple_animal/bot/bot.dm

```diff
@@ -130,6 +130,16 @@
 		return "Inactive"
 	return "[mode]"
 
+/**
+ * Returns a string of flavor text for emagged bots as defined by policy.
+ */
+/mob/living/simple_animal/bot/proc/get_emagged_message()
+	var/emagged_message = "You are a malfunctioning bot! Disrupt everyone and cause chaos!"
+	var/policy = get_policy(ROLE_EMAGGED_BOT)
+	if (policy)
+		emagged_message = policy
+	return emagged_message
```

Comment:
This whole thing should be `get_policy(ROLE_EMAGGED_BOT) || "Your custom text here"`